webviews/src/shared/OdcSetupSteps.svelte

```diff
@@ -7,18 +7,28 @@
         padding: 10px;
     }
 
-    .codeSnippet {
+    code {
         color: orange;
+        display:inline-block;
         font-weight: bold;
     }
 </style>
 
 <div id="odcSetupSteps">
-    This tool requires a component be included and initialized in your application (similar to RALE). The easiest way to do this is:
-    <ul>
-        <li>Include the following comment in either your Scene or main.brs file (if including in main.brs, be sure to add after your roSGScreen screen.show() call)<br><span class="codeSnippet">' vscode_rdb_on_device_component_entry</span><br></li>
-        <li>Make sure your launch.json configuration has<br><span class="codeSnippet">"injectRdbOnDeviceComponent": true</span> included in it</li>
-    </ul>
-    The extension can copy the files automatically for you so there's no need to handle that part. If you are still having issues even with these steps, check to make sure you're seeing this line in your device logs <span class="codeSnippet">[RTA][INFO] OnDeviceComponent init</span>
+    Could not connect to the on-device component. Install steps:
+
+    <ol>
+        <li>
+            Enable the following property in <b><i>launch.json</i></b>:
+            <pre><code>"injectRdbOnDeviceComponent": true</code></pre>
+        </li>
+        <li>
+            Add this comment to <code>function main()</code> after the call to <code>screen.show()</code>
+            <pre><code>' vscode_rdb_on_device_component_entry</code></pre>
+        </li>
+    </ol>
+
+    The ODC is properly installed if you see this log output during channel launch:
```

Comment:
The ODC component?

Reply:
Odc means on device component. So you want it to say "on-device-component component"? 😜

I'd rather spell it out in that case.

Reply:
Maybe `The on-device component is properly installed if you see this log output during channel launch:`